packages/financial-templates-lib/test/price-feed/CreatePriceFeed.js

```diff
@@ -299,6 +299,67 @@ contract("CreatePriceFeed.js", function(accounts) {
     assert.isTrue(balancerFeed instanceof BalancerPriceFeed);
   });
 
+  it("Create token price feed for Uniswap", async function() {
+    const collateralTokenAddress = "0x0000000000000000000000000000000000000001";
+    const config = {
+      type: "uniswap",
+      uniswapAddress,
+      twapLength,
+      lookback
+    };
+
+    const constructorParams = {
+      expirationTimestamp: ((await web3.eth.getBlock("latest")).timestamp + 1000).toString(),
+      withdrawalLiveness: "1000",
+      collateralAddress: collateralTokenAddress,
+      finderAddress: Finder.address,
+      tokenFactoryAddress: TokenFactory.address,
+      priceFeedIdentifier: web3.utils.utf8ToHex("ETH/BTC"),
+      syntheticName: "Test UMA Token",
+      syntheticSymbol: "UMATEST",
+      liquidationLiveness: "1000",
+      collateralRequirement: { rawValue: toWei("1.5") },
+      disputeBondPct: { rawValue: toWei("0.1") },
+      sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
+      disputerDisputeRewardPct: { rawValue: toWei("0.1") },
+      minSponsorTokens: { rawValue: toWei("1") },
+      timerAddress: Timer.address
+    };
+
+    const emp = await ExpiringMultiParty.new(constructorParams);
+
+    const uniswapFeed = await createTokenPriceFeedForEmp(logger, web3, networker, getTime, emp.address, config);
```

Comment:
These all test the behavior of `createTokenPriceFeedForEmp` depending on the inputted `config` object. If `config` is unspecified, it should default to Medianizer